bin/exporter

```diff
@@ -28,4 +28,5 @@ AuroraBootstrapper::Exporter.new( client: client,
                            export_bucket: bukkit,
                       blacklisted_tables: blacklisted_tables,
                       whitelisted_tables: whitelisted_tables,
-                      blacklisted_fields: blacklisted_fields ).export!
\ No newline at end of file
+                      blacklisted_fields: blacklisted_fields,
+                      notifier: Notifier.new ).export!
```

Comment:
What's the thinking behind passing this into the Exporter as a param? Since there aren't multiple different notifiers, can we init this in the Exporter?

Reply:
For testing purpose

Reply:
If I pass it in I can mock the notifier in the test

Reply:
If I create a Notifier inside exporter, then I have no way to mock this class's method in unit tests

Reply:
Is is not considered best practice to design code around testing. How can I help get this addressed?